detox/test/detox.config.js

```diff
@@ -97,57 +149,41 @@ const config = {
       }
     },
     "android.emu.debug": {
-      binaryPath: "android/app/build/outputs/apk/fromBin/debug/app-fromBin-debug.apk",
-      utilBinaryPaths: [
-        "./cache/test-butler-app.apk"
-      ],
-      build: "cd android && ./gradlew assembleFromBinDebug assembleFromBinDebugAndroidTest -DtestBuildType=debug && cd ..",
-      type: "android.emulator",
-      device: {
-        avdName: "Pixel_API_28"
+      device: "android.emulator",
+      app: {
+        type: "android.apk",
+        binaryPath: "android/app/build/outputs/apk/fromBin/debug/app-fromBin-debug.apk",
+        utilBinaryPaths: [
+          "./cache/test-butler-app.apk"
+        ],
+        build: "cd android && ./gradlew assembleFromBinDebug assembleFromBinDebugAndroidTest -DtestBuildType=debug && cd ..",
       },
-      launchArgs,
     },
     "android.emu.release": {
-      binaryPath: "android/app/build/outputs/apk/fromBin/release/app-fromBin-release.apk",
-      utilBinaryPaths: [
-        "./cache/test-butler-app.apk"
-      ],
-      build: "cd android && ./gradlew assembleFromBinRelease assembleFromBinReleaseAndroidTest -DtestBuildType=release && cd ..",
       type: "android.emulator",
       device: {
         avdName: "Pixel_API_28"
       },
+      build: "cd android && ./gradlew assembleFromBinRelease assembleFromBinReleaseAndroidTest -DtestBuildType=release && cd ..",
+      binaryPath: "android/app/build/outputs/apk/fromBin/release/app-fromBin-release.apk",
```

Comment:
'this here on purpose, in order to test config overrides?

Reply:
Yeah — but maybe it is too weird to keep it this way.